### DISCUSSION TITLE
Compressed model gltf cannot be used offline

### DISCUSSION THREAD

**Discussion by FEJackFly** [visgl/deck.gl#8038] [Q&A]
DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://unpkg.com/@loaders.gl/draco@3.4.4/dist/draco-worker.js' failed to load.

How to use offline

**Comment by ibgreen** [comment]
You should be able to set `worker: false` in [layer.props.loadOptions](https://loaders.gl/docs/modules/core/api-reference/loader-options), or copy the worker to your offline server and provide a `workerUrl` option for the draco loader to make it load from your server.
